Cour-de-cassation/label
# adding checklist from annotationReports in statistics collection

## Issue description :
adding checklist from annotationReports collection to statistics collection for saving after in sder for data team
## Describe your changes :

## How to test :
annotate a document with checlist (mise en doute) and export it, go to statistics collection and see if it saved the (checkList)
## Checklist before requesting a review
